### PR TITLE
- Corrected usage of `grepl` in an if-statement in `validateIsOfType`

### PR DESCRIPTION
**Pull request by PavelBal**
This was causing test fails when locally performing "check" for ospsuite-r.

### CODE REVIEW

File: R/validation-type.R [hunk 1]
```diff
@@ -81,7 +81,7 @@ validateIsOfType <- function(object, type, nullAllowed = FALSE) {
   callStack <- as.character(sys.call(-1)[[1]])
 
   # Object name is one frame further for functions such as ValidateIsNumeric
-  if ((length(callStack) > 0) && grepl(pattern = "validateIs", x = callStack)) {
+  if ((length(callStack) > 0) && any(grepl(pattern = "validateIs", x = callStack))) {
```
Comment (PavelBal) [inline R/validation-type.R:84]: `grepl` returns a list of booleans if `callStack` is larger than 1, but we need a single boolean here.

**Comment by PavelBal** [comment]
@Yuri05  `test-coverage / test-coverage (pull_request) ` fails because it cannot install R 4.2.2 (for whatever reason...)

`R-CMD-check / windows-latest (devel) (pull_request) ` because it uses R 4.3 (for whatever reason) which has not been released yet and maybe something has been changed there.

**Comment by Yuri05** [comment]
>R-CMD-check / windows-latest (devel) (pull_request) because it uses R 4.3 (for whatever reason) which has not been released yet and maybe something has been changed there.

I would suggest to remove all `devel` checks from the action

**Comment by Yuri05** [comment]
I restarted the test coverage and it worked - was just a temporary glitch for sure.

**Comment by PavelBal** [comment]
@Yuri05 removed checks for `devel` version of R.

Ubuntu job fails with

`Error: Failed to get R 4.1.3: Failed to get R 4.1.3: Failed to install R: Error: The process '/usr/bin/sudo' failed with exit code 1`

No idea why.

**Review by Yuri05 — APPROVED** [review]

**Comment by Yuri05** [comment]
>Ubuntu job fails with

@PavelBal Restarted the job with success. Again, seems to be a temporary problem of GitHub(?)

**Pull request merged**